src/inputs/textArea.jsx

```diff
@@ -112,7 +113,7 @@ class TextArea extends React.Component {
             const textArea = ReactDOM.findDOMNode(this.textArea);
 
             const autoResize = setInterval(() => {
-                if (value || this.textArea.value) {
+                if (value || get(this, 'textArea.value')) {
```

Comment:
I am curious if this is just being extra cautious, or if there was actually an observed case of `this.textArea` being null or undefined.  It's not a problem to make this change; I'm just curious if it was necessary or strictly a precaution.

Reply:
We're receving this error due to the setInterval, I doscovered it when working with person note reminders, that's not part of the scope but I decided to add this little update to avoid the error.
![image](https://user-images.githubusercontent.com/8116485/88503328-02ab1800-cf97-11ea-870e-6b7627b7110e.png)